tools/ormolu.sh

```diff
@@ -2,7 +2,7 @@
 
 cd "$( dirname "${BASH_SOURCE[0]}" )/.."
 
-stack exec -- which ormolu >/dev/null || ( echo "please run stack install ormolu."; exit 1 )
+which ormolu >/dev/null || ( echo "please run 'stack install ormolu' and ensure ormolu is on your PATH."; exit 1 )
```

Comment:
not tested:

```suggestion
ORMOLU_VERSION=0.0.3.1
ormolu -v >/dev/null 2>&1 | grep -q $ORMOLU_VERSION || ( echo "please install ormolu $ORMOLU_VERSION (eg., run 'stack install ormolu' and ensure ormolu is on your PATH.)"; exit 1 )
```

Reply:
This means we have to bump the ormolu version in this file every time we bump it in our stack.yaml? If there's a way to avoid that, that would be nice. Otherwise, we should at least leave a comment in the stack.yaml.

Reply:
good idea: d6c1ef12e47911709b1e37157c95e974ee5bd8b6

Reply:
(sorry about the perl dep, if anybody wants to translate this to sed, i'll start a translation table for next time.  :))

Reply:
oops: 87f8cc2fdc0d501d10be79db6476affe64e17470

Reply:
:+1: for the ormolu version checking (and adding to stack.yml)